notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaAttribute.java

```diff
@@ -114,4 +117,9 @@ public void addValue(int index, String value) {
         values.add(index, value);
     }
 
+    @Override
+    public String toString() {
+        return "JpaAttribute [id=" + id + ", name=" + name + ", values=" + values + "]";
```

Comment:
would [`org.apache.commons.lang3.builder.ToStringBuilder`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html) be a more effective way to generate this string?